packages/editor/src/core/all-plugins.tsx

```diff
@@ -0,0 +1,576 @@
+import IconBlanksDragAndDrop from '@editor/editor-ui/assets/plugin-icons/icon-blanks-dnd.svg?raw'
+import IconBlanksTyping from '@editor/editor-ui/assets/plugin-icons/icon-blanks-typing.svg?raw'
+import IconBox from '@editor/editor-ui/assets/plugin-icons/icon-box.svg?raw'
+import IconDropzones from '@editor/editor-ui/assets/plugin-icons/icon-dropzones.svg?raw'
+import IconEquation from '@editor/editor-ui/assets/plugin-icons/icon-equation.svg?raw'
+import IconGeogebra from '@editor/editor-ui/assets/plugin-icons/icon-geogebra.svg?raw'
+import IconH5p from '@editor/editor-ui/assets/plugin-icons/icon-h5p.svg?raw'
+import IconHighlight from '@editor/editor-ui/assets/plugin-icons/icon-highlight.svg?raw'
+import IconImage from '@editor/editor-ui/assets/plugin-icons/icon-image.svg?raw'
+import IconInjection from '@editor/editor-ui/assets/plugin-icons/icon-injection.svg?raw'
+import IconTextArea from '@editor/editor-ui/assets/plugin-icons/icon-input-exercise.svg?raw'
+import IconMultimedia from '@editor/editor-ui/assets/plugin-icons/icon-multimedia.svg?raw'
+import IconSpoiler from '@editor/editor-ui/assets/plugin-icons/icon-spoiler.svg?raw'
+import IconTable from '@editor/editor-ui/assets/plugin-icons/icon-table.svg?raw'
+import IconText from '@editor/editor-ui/assets/plugin-icons/icon-text.svg?raw'
+import IconVideo from '@editor/editor-ui/assets/plugin-icons/icon-video.svg?raw'
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+
+import { Instance } from '@/fetcher/graphql-types/operations'
+import { getLoggedInData } from '@/helper/feature-i18n'
+
+// Need this tiny interface, because the mergeDeepObject seems to
+// screw up the types of our i18n strings
+type PluginStrings = Record<
+  EditorPluginType,
+  {
+    title: string
+    description: string
+  }
+>
+
+const germanPluginStrings = getLoggedInData(Instance.De).strings.editor
+  .plugins as unknown as PluginStrings
+
+const englishPluginStrings = getLoggedInData(Instance.En).strings.editor
+  .plugins as unknown as PluginStrings
+
+const getPluginNameAndDescription = (
+  locale: 'de' | 'en',
+  pluginType: EditorPluginType
+) => {
+  let name: string
+  let description: string
+  if (locale === 'de') {
+    name = germanPluginStrings[pluginType]?.title
+    description = germanPluginStrings[pluginType]?.description
+  } else if (locale === 'en') {
+    name = englishPluginStrings[pluginType]?.title
+    description = englishPluginStrings[pluginType]?.description
+  } else {
+    throw new Error('Invalid locale')
+  }
+
+  if (!name || !description) {
+    throw new Error(
+      'Missing plugin name or description for plugin type' + pluginType
+    )
+  }
+
+  return {
+    name,
+    description,
+  }
+}
+
+const getInternationalizedPluginStrings = (type: EditorPluginType) => ({
+  de: getPluginNameAndDescription('de', type),
+  en: getPluginNameAndDescription('en', type),
+})
+
+export const AllPlugins = [
```

Comment:
```suggestion
export const pluginDescriptions = [
```

Reply:
Can you do `typeof allPlugins = Record<PluginType, PluginTypeDescription>` => So a dict instead of a list. Can you also export PluginType as a enum?